.prettierrc.yaml

```diff
@@ -0,0 +1,6 @@
+printWidth: 100
+singleQuote: true
+semi: false
+bracketSpacing: false
+tabWidth: 4
+arrowParens: 'always'
```

Comment:
So, inside the check-dependencies script we explicitly check that all the prettier rc files are the same for each project in the mono-repo. This particular file however is not checked to be the same as them, so it's kinda breaking this promise that all the prettier are the same. What are your thoughts?

Reply:
@bendvc I think it's not the end of the world if this root prettier config is not exactly the same with the configs in the projects, because this is not something we will ship to customers, the only effect it has is the code in the `scripts` folder, which is internal.

I want to avoid writing code to keep this in sync, feel like it's not worth the effort and complexity.

The PR description had a false promise, i updated it.

(The reason i make this PR is to unblock Sejal's PR https://github.com/SalesforceCommerceCloud/pwa-kit/pull/269.)

Reply:
I manually reformatted Sejal's PR and merged it, so there is no urgency to merge this PR.